### ISSUE TITLE
Only /status works & Adding another subproject

### ISSUE THREAD

**Issue by danks1234** [adrianhurt/play-multidomain-seed#9]
Hello Adrian,
I'm really thankful for what you've done here, it's a really nice seed and, althought at the start it was a little infuriating, I've had my fun just setting it up.
Before I forget, I was having some trouble with iheart:ficus not being found, so I added this resolver "Resolver.jcenterRepo," and changed the version to: "com.iheart" % "ficus_2.11" % "1.3.0", might come in handy.

So, once the seed finally decided to start I went directly to myweb.com:9000 and was met with no action found, only /status from common seems to work. This happens whether I use localhost or whatever other domain, I was wondering if I missed/messed up something.
But I also don't see a routes files in the main conf folder, you do say that there's supposed to be one while the website is in use, and what it does is pick up and mix all the other routes, but that seems missing.
Any ideas?

Also, I added another subproject to the mix (worry not, the routes didn't work before I decided to add this), your explanation was very thorough and everything seems like it's on its place, so thanks again for doing all this.

Saludos,
Daniel

![image](https://cloud.githubusercontent.com/assets/16642619/20028959/53a317f2-a31e-11e6-8a0e-bc558db48f11.png)
![image](https://cloud.githubusercontent.com/assets/16642619/20028964/663003da-a31e-11e6-91a5-7209ad580ca4.png)
![image](https://cloud.githubusercontent.com/assets/16642619/20028968/72da39c0-a31e-11e6-8218-1b01ec5f92bf.png)
![image](https://cloud.githubusercontent.com/assets/16642619/20028972/8dfacc88-a31e-11e6-8857-8fb0640854fb.png)

**Comment by danks1234** [comment]
Seems to have loaded without issues (after adding client)

![image](https://cloud.githubusercontent.com/assets/16642619/20029000/434bb0fc-a31f-11e6-989c-e3ca11f28b9a.png)

**Comment by adrianhurt** [comment]
Hi @danks1234.

It's strange, I don't know what to say. I've just downloaded the zip from the repository and simply:
    $ cd /path/to/play-multidomain-seed-master/
    $ activator
    [play-multidomain-seed] $ run

And then it works for `localhost:9000`, `myweb.com:9000`, `www.myweb.com:9000` and `admin.myweb.com:9000`.

I haven't had any issues with `ficus`. Maybe your IDE or the environment configuration? I've tested without any IDE on a Mac with macOS Sierra.

Saludos a ti también!

**Comment by danks1234** [comment]
yeah, checks out, running on Windows10 and IntelliJ

**Comment by adrianhurt** [comment]
Hi, then did it work? I'm closing the issue, but feel free to reopen.

BTW: I've just updated the ficus dependency as you said.

**Comment by dmonteroh** [comment]
Sorry about the super late reply, I have been busy with my day job. I will download the last version and try again, it didn't work last time which was kind of sad, keep this closed if you will, I'll let you know eventually how it goes.

Thanks a lot again! Saludos.

Edit: You know, I downloaded the multidomain with auth and that one worked just straight out of the bat, no issues. I will just figure out how to have the sessions shared between the subdomains and I guess it's just migrating to that seed

Edit 2: Aditionally I did try to get this one running, I have no idea what's going on tho...
![image](https://cloud.githubusercontent.com/assets/16642619/20955023/aa65a88e-bc1e-11e6-9025-c98f55c519ff.png)

Edit 3: I have no idea what's going on... it's working now, keep it closed and thanks a bunch, yet again
![image](https://cloud.githubusercontent.com/assets/16642619/20957874/279c34a0-bc31-11e6-9cad-39ee57b110d2.png)

Edit 4: Me rindo, estoy demasiado frustrado, no entiendo la mitad de las cosas que tengo que hacer, quiero implementar el auth que hiciste en el multidomain, pero que sea un solo auth para toda la pagina y nada, modificar el multidomain con auth que ya existe parece imposible. Implementando desde cero en este seed que ahora si funciona, con play-authenticate basado en un seed de esos basicos, nada, basado en tus otros authenticate, nada. No sere el mejor programador del mundo, pero nunca pense que fuera tan malo, nunca me habia sentido tan frustrado y nunca me habia costado tanto entender lo que esta pasando, leo tu documentacion y lo entiendo pero a la hora de hacer un cambio, cero. Y que este en scala me mata un poquito mas, que triste sentirse asi.
